app/graph/nodes/tool/navigation_node.py

```
@@ -0,0 +1,316 @@
+def _search_place(site_id: int, query: str) -> Optional[Dict[str, Any]]:
+    """Core /places/search 호출. threshold 미달이면 None."""
+    try:
+        resp = requests.get(
+            f"{CORE_BASE_URL}/internal/v1/places/search",
+            params={"siteId": site_id, "q": query, "threshold": _SEARCH_THRESHOLD},
+            timeout=5.0,
+        )
+        resp.raise_for_status()
+        data = resp.json()
+        return data if isinstance(data, dict) else None
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)